formstack-baton-requests/build.sbt

```diff
@@ -0,0 +1,41 @@
+
+name := "formstack-baton-requests"
+
+version := "0.1"
+
+scalaVersion := "2.12.8"
+val circeVersion = "0.13.0"
+val amazonSdkVersion = "1.11.755"
+
+libraryDependencies ++= Seq(
+  "com.amazonaws" % "aws-lambda-java-core" % "1.2.0",
+  "com.amazonaws" % "aws-java-sdk-s3" % amazonSdkVersion,
+  "com.amazonaws" % "aws-java-sdk-lambda" % amazonSdkVersion,
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.0",
+  "org.jlib" % "jlib-awslambda-logback" % "1.0.0",
+  "io.circe" %% "circe-core" % circeVersion,
+  "io.circe" %% "circe-parser" % circeVersion,
+  "io.circe" %% "circe-generic" % circeVersion,
+  "org.scalatest" %% "scalatest" % "3.0.5" % "test"
+)
+
+scalacOptions += "-Ypartial-unification"
+
+addCompilerPlugin(
+  "org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full
```

Comment:
minor: what is this for ?

Reply:
I've removed this now. It gave me access to @JsonCodec in circe but I don't need it anymore